src/backend/joanie/core/serializers/client.py

```diff
@@ -240,12 +240,26 @@ class NestedOrderSerializer(serializers.ModelSerializer):
     id = serializers.CharField(read_only=True, required=False)
     course = CourseLightSerializer(read_only=True, exclude_abilities=True)
     organization = OrganizationSerializer(read_only=True, exclude_abilities=True)
+    title = serializers.SerializerMethodField()
+    learner_name = serializers.SerializerMethodField()
```

Comment:
should be called `owner_name` since you get it with `instance.owner.get_full_name()`?

Reply:
I'll call it `owner` to stick to model.